pkgs/development/python-modules/flash-attn/default.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  einops,
+  ninja,
+  setuptools,
+  symlinkJoin,
+  torch,
+}:
+
+let
+  inherit (torch) cudaCapabilities cudaPackages cudaSupport;
+  inherit (cudaPackages) backendStdenv;
+
+  cutlass = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "cutlass";
+    tag = "v4.1.0";
+    hash = "sha256-ZY+6Tg/CC6fqvU764k6QNudYDpY+s8OQklG+1aXQuns=";
+  };
+in
+buildPythonPackage rec {
+  pname = "flash-attention";
+  version = "2.8.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Dao-AILab";
+    repo = "flash-attention";
+    tag = "v${version}";
+    hash = "sha256-RrayQOxQzGJMQK5jmMziR59p8CTF8mpEyJsqzouEW1s=";
+  };
+  postPatch = ''
+    rmdir csrc/cutlass
```

Comment:
Why do you need to do this? Can't you just set `fetchSubmodules = true;` in the call to `fetchFromGitHub` and update the hash?

If not, please add a description in a comment on why this is necessary.